polaris-react/src/components/Box/Box.tsx

```diff
@@ -18,28 +20,12 @@ import styles from './Box.scss';
 
 type Element = 'div' | 'span' | 'section' | 'legend' | 'ul' | 'li';
 
+type LineStyles = 'solid' | 'dashed' | 'none';
```

Comment:
Do we use dashed anywhere? Why do we need none here?

Reply:
`dashed` is used on some `<IndexTable>` filters. It's not super common but I want to make sure it is available. I don't know if we need `none` we can set not set the `borderWidth` and it does the same thing. Happy to remove `none`.